src/licensedcode/data/licenses/oculus-sdk.yml

```diff
@@ -0,0 +1,8 @@
+key: oculus-sdk
+short_name: Oculus SDK License Agreement 
+name: Oculus SDK License Agreement
+category: Copyleft Limited
```

Comment:
@DennisClark would this be the right category?

Reply:
@DennisClark gentle ping :)

Reply:
Copyleft Limited is fine here.